src/libcore/thread.cpp

```diff
@@ -13,6 +11,11 @@
 // Required for native thread functions
 #if defined(__LINUX__)
 #  include <sys/prctl.h>
+#  ifdef _SC_NPROCESSORS_CONF
+#    define NPROCESSORS_COUNT sysconf(_SC_NPROCESSORS_CONF)
```

Comment:
I still don't understand why this has to be declared at the top. It always worked on linux, no?

Reply:
Right, we need to include `#include <unistd.h>` then.